### PR TITLE
Adding msrest and msrestazure

### PR DESCRIPTION
**Pull request by ivoflipse**
In order to fix the recipes for azure-mgmt-* we need msrestazure, which in turn needs mrest. See https://github.com/conda-forge/azure-mgmt-storage-feedstock/pull/5

### CODE REVIEW

**Comment by ivoflipse** [comment]
Interesting that on Travis it accepts the `pip intall` command, where on CircleCI it complains it can't find pip

**Comment by ivoflipse** [comment]
And now I get hit with a random error on CircleCI **sigh**

File: recipes/adal/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+test:
+  imports:
+    - adal
+
+about:
+  home: https://github.com/AzureAD/azure-activedirectory-library-for-python
+  license: MIT
+  license_family: MIT
+  summary: The ADAL for Python library makes it easy for python application to authenticate to Azure Active Directory (AAD) in order to access AAD protected web resources.
```
Comment (CJ-Wright) [inline recipes/adal/meta.yaml]: Please break this up over multiple lines

Comment (ivoflipse) [reply recipes/adal/meta.yaml]: I had that, but my syntax was invalid and then I couldn't find the spec fast enough, so I reverted it. I'll try it again

**Comment by ivoflipse** [comment]
I think they could all be noarch yes and switched back to `python -m pip install --no-deps --ignore-installed .`

**Comment by CJ-Wright** [comment]
These are all noarch packages and thus only need to pass on circle.

**Comment by ivoflipse** [comment]
That’s good to know for the future

**Comment by CJ-Wright** [comment]
It is a good incentive (on all sides) and was invaluable when travis had a ton of OSX problems

**Pull request merged**